content/en/docs/release_notes/austen.md

```diff
@@ -10,7 +10,8 @@ Brief summary of the notable release contents.
 
 {{% releasenotes/upgrade %}}
 
-<!-- ## Breaking changes -->
+## Breaking changes
+* #7479: [`config.admin.updatesEnabled`](/docs/developer_guide/project/configuration/project_configuration/#admin) no longer has its default behavior controlled by [`config.bindProcessHandlers`](/docs/developer_guide/project/configuration/project_configuration/#bindprocesshandlers). If your project unit tests still directly set `bindProcessHandlers` you should follow [this guide](/docs/updates/2022_08_26_update_unit_tests_with_test_utils) to update your unit tests to use `@axway/api-builder-test-utils` which will control `updatesEnabled` and `bindProcessHandlers` under the hood.
```

Comment:
**suggestion**: I don't like how we expose the internal trick with "bindProcessHandlers" to our users, and it isn't clear to our users what the relationship is between these two flags, or why they need to upgrade.

**question**: Have we added "updatesEnabled" to the Project configuration doc yet?

```suggestion
* #7479: [`config.admin.updatesEnabled`](/docs/developer_guide/project/configuration/project_configuration/#admin) is enabled by default, and should be explicitly disabled for unit tests (similar to [`bindProcessHandlers`](/docs/developer_guide/project/configuration/project_configuration#bindprocesshandlers)). Unit tests should be updated to use the [`@axway/api-builder-test-utils`](https://www.npmjs.com/package/@axway/api-builder-test-utils), which disables `updatesEnabled` and `bindProcessHandlers` by default. If your project unit tests still directly set `bindProcessHandlers`, then you should follow [this guide](/docs/updates/2022_08_26_update_unit_tests_with_test_utils) to upgrade them.
```